java/openjdk8-powerpc/Portfile

```diff
@@ -0,0 +1,324 @@
+                    sha256  c447e801b2d66823af320710738c44d51e6d62496569d39bfbe5d0dd822bd8c5 \
+                    size    60233820
+
+# Generic patches:
+patchfiles          0001-Fix-timebomb-bug-in-GenerateCurrencyData.patch \
+                    0002-Ignore-__NSAutoreleaseNoPool-errors.patch \
+                    0003-os_perf_bsd-allow-to-compile-with-ZERO-CPU.patch \
+                    0004-Added-macosx-bin-zero-jvm.cfg.patch \
+                    0005-ExtendedOptionsImpl.c-define-TCP_KEEPINTVL-and-TCP_K.patch \
+                    0006-Revert-8211301-macos-support-full-window-content-opt.patch \
+                    0007-Provide-definition-of-__private_extern__-where-undef.patch
+
+# If we build with Xcode gcc, we do not need any C11.
+if {![string match gcc-4.* ${configure.compiler}]} {
+    patchfiles-append \
+                    0008-8181503-Can-t-compile-hotspot-with-c-11.patch \
+                    0009-Disable-C-11-warnings.patch
+}
+
+# PowerPC-specific patches:
+platform powerpc {
+    patchfiles-append \
+                    1001-os_bsd_zero.cpp-implement-sync_val_compare_and_swap_.patch \
+                    1002-CoreLibraries.gmk-do-not-hardcode-arch.patch \
+                    1003-java_md_macosx-fix-broken-ObjC-code.patch \
+                    1004-os_bsd-add-ppc64-case.patch \
+                    1005-jniTypes_ppc-downgrade-error-to-warning.patch \
+                    1006-os_bsd.cpp-fix-vm_statistics.patch \
+                    1007-Awt2dLibraries-add-fnested-functions.patch \
+                    1008-java_md_macosx.c-define-preferredJVM.patch \
+                    1009-os_bsd.cpp-allow-headless-on-Apple.patch \
+                    1010-Makefile-fix-LP64-define.patch \
+                    1011-No-dtrace-for-now.patch \
+                    1012-toolchain.m4-hack-around-a-broken-Xcode-version-dete.patch \
+                    1013-os_bsd.cpp-fix-running-VM.patch \
+                    1014-portconfig-try-fix-broken-pipe.patch \
+                    1015-Some-hacks-to-fix-the-build-for-powerpc.patch
+}
+
+# Active libnet breaks linking of libnio which should link
+# to JDK’s own internal libnet.
+# libjpeg-turbo: https://trac.macports.org/ticket/67123
+conflicts_build     libjpeg-turbo libnet
+
+# If you want to use an alternative JDK for bootstrap,
+# change these values accordingly:
+depends_build       port:openjdk7-bootstrap
+configure.args      --with-boot-jdk=/Library/Java/JavaVirtualMachines/openjdk7-bootstrap
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:bash \
+                    port:gmake \
+                    path:bin/pkg-config:pkgconfig
+
+depends_lib-append  port:fontconfig \
+                    port:freetype \
+                    port:libffi \
+                    port:libiconv
+
+set tpath           /Library/Java
+
+use_xcode           yes
+
+if {${configure.build_arch} in [list i386 ppc]} {
```

Comment:
Why is i386 in here?

Reply:
I normally use a canonical expression for something which is required on 32-bit, regardless of whether a specific arch is actively supported. (It can’t hurt, but will be helpful if someone restores i386 support.)
However, I don’t really care about i386, at least due to time constraints, so can drop this once back to the machine tomorrow.

Reply:
Please do.

Reply:
Ah, I can do it in a way that i386 will still work, but without having it mentioned. Fixed.